setup.py

```diff
@@ -93,6 +101,8 @@
     'License :: OSI Approved :: Apache Software License',
 ]
 
+BUILD_WITH_BORING_SSL_ASM = os.environ.get('GRPC_BUILD_WITH_BORING_SSL_ASM', False)
```

Comment:
Can we make it default on and run a distribution test?

Reply:
turned it to default .Is the distribution test run as part of CI?  Or do I need to run it by hand.